test/proto2pb/test_all_types.proto

```diff
@@ -116,13 +116,6 @@ message ExampleType {
   extensions 100 to max;
 }
 
-// Package scoped extensions.
-extend ExampleType {
-  optional ExampleType nested_example = 100;
-  optional int32 int32_ext = 101;
-  optional google.protobuf.Int32Value int32_wrapper_ext = 102;
-}
-
 // Message scoped extensions.
```

Comment:
Could we also move this to a separate file? That would [more accurately test the message scoped cases here](https://github.com/google/cel-go/blob/b8c669d93b21fc766a05c7199be531f439572fcc/common/types/pb/file_test.go#L48-L55)

Reply:
I intentionally split the extensions to cover the case where they're spread across the same file and a separate file. I can definitely add a message extension into the test_extensions proto though. Does that sound reasonable?

Reply:
I see, what you proposed sounds good to me.

Reply:
PTAL. It looked like the `protos.go` and `protos_test.go` files weren't included in the Bazel build and there were some additional issues there with how extensions were being resolved at runtime. Additionally, some issues cropped up with ensuring that extensions were hermetically sealed between `Db.Copy()` calls which have been addressed as well.